knowledge-base/shivammathur/setup-php/action-security.yml

```diff
@@ -3,7 +3,7 @@ github-token:
   environment-variable-name: COMPOSER_TOKEN
   permissions:
     contents: read
-    contents-reason: to access matching refs GitHub API, clone the repos, and get release assets.
+    contents-reason: to access matching refs of GitHub API.
```

Comment:
Can you please add link to reason
`contents-reason: to clone the repos # https://place in the repo where reason is mentioned`